STRATEGIES.md

```diff
@@ -0,0 +1,23 @@
+# How do I push changes to github?
+[Story](https://www.pivotaltracker.com/story/show/153708395)
+------------------------------------------------------------
+
+# How do I make a pull request?
+step 1: go to github
+Step 2: click the "pull requests" tab in top left corner (https://github.com/altereagle/trinity-carpool/pulls)
+Step 3: click the green "new pull request" button
+step 4:change the "base" and "compare" sections to what is needed
+Step 5:
+step 6:
```

Comment:
need to either put in these steps or take these out

Reply:
Ya we should take them out

Reply:
step 5 and 6